utils/clientProcessing.js

```diff
@@ -0,0 +1,30 @@
+
+/**
+ * Given an array of clients from the MindBody API and an array of client emails
+ * from the database associated with a specific job, two arrays are returned. The first
+ * being clients from API but not in the database and the second being clients in
+ * the database but not from API.
+ * 
+ * @param {*} apiClients array of clients from api call.
+ * @param {*} storedClientsEmails array of emails of clients from database.
+ * @param {*} jobId job_id of the class the clients are attending.
+ */
+async function processClients(apiClients, storedClientsEmails, jobId) {
+    const apiClientEmails = await Promise.all(apiClients.map(async function (client) {
```

Comment:
actually right here it won't be necessary to have an async function because the content inside is just a simple return. You can change the syntax to an arrow function: 

```
const apiClientEmails = await Promise.all(apiClients.map(client => client.email));
```

Same for the rest of these lines that uses async functions when the body is just a simple return.